src/tokenizer/stop_word_filter/mod.rs

```diff
@@ -31,15 +45,28 @@ impl StopWordFilter {
         }
     }
 
-    fn english() -> StopWordFilter {
-        let words: [&'static str; 33] = [
+    /// Create a `StopWorldFilter` for the named language
+    pub fn english() -> Self {
+        const WORDS: &[&str] = &[
```

Comment:
Add one line to explain the origin of this list. (Copied from Lucene).

Reply:
Done.